package.json

```diff
@@ -20,7 +20,8 @@
     "generate-typescript-node": "node ./lib/CLI/index.js https://faker.graphqleditor.com/a-team/olympus/graphql examples/typescript-node/src --ts --n -g examples/typescript-node/zeus.graphql --apollo",
     "run-example-typescript-node": "./run-example.sh examples/typescript-node",
     "run-example-javascript-node": "./run-example.sh examples/javascript-node",
-    "lint": "ttsc  && eslint \"./src/**/*.{ts,js}\" --quiet --fix"
+    "lint": "ttsc  && eslint \"./src/**/*.{ts,js}\" --quiet --fix",
+    "prepare": "npm run build"
```

Comment:
Please either remove `prepare` or fix the CI actions to reflect changes here.

Reply:
I’ve added this in so I could use my git fork in the meantime. I don’t mind removing this, however I’m not sure I understand what you mean by fix the CI action to reflect changes here, do you mind elaborating please ?